generator/template/pages/holder/components/CredentialDetails/CredentialDetails.tsx

```diff
@@ -23,13 +23,13 @@ export const CredentialDetails: FC<CredentialDetailsProps> = ({
   practitionerName,
   qrCode,
 }) => (
-  <S.CredentialDetailsContainer justifyContent='space-between' direction="row">
-    <S.CredentialDetailsCard justifyContent='space-between' gap={48}>
+  <S.DetailsContainer justifyContent='space-between' direction="row">
+    <S.DetailsCard justifyContent='space-between' gap={48}>
       <Box gap={8}>
         <Typography variant='h4'>{patientName}</Typography>
       </Box>
 
-      <S.MedicationDetailsContainer gap={42}>
+      <S.DataContainer gap={42}>
```

Comment:
@vitalyrudenko-affinidi Didn't you leave this prefix of Medication on purpose? I remember you mentioned the template would default to Healt

Reply:
I makes sense, but let's hear from @vitalyrudenko-affinidi

Reply:
@carlos-affinidi @vitalyrudenko-affinidi Did we come to a conclusion or can I just merge?

Reply:
If this is a generic component, then it makes sense to name it generic. If it's not generic and only used in one use case, I'd be more careful.

But this is not a big deal at the moment so we can merge if you think it won't confuse us later.

Reply:
Yeah, this is on the health app, which is the base of everything if it's not overridden. If we added gaming into this and placed no overrides on the holder components it would show up in the use cases with "health" specific naming which isn't great imo